src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -163,16 +163,17 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
 
             var sections = await descriptionService.ToDescriptionGroupsAsync(workspace, semanticModel, token.SpanStart, symbols.AsImmutable(), cancellationToken).ConfigureAwait(false);
 
+            ImmutableArray<TaggedText> parts;
```

Comment:
There were already four local variables, all `ImmutableArray<TaggedText> parts`, but all scoped within the if blocks. Using four (or five) `out var` variables would have been difficult and somewhat artificial, since they would all have to have different names. I just reused one local variable. Also, I did not declare it in the first `TryGetValue` call, to keep them "symmetrical", not make the first `if` block special.